src/engine-actions/translate-swap.js

```diff
@@ -144,13 +144,7 @@ async function translateSwap (takerAddress, swapHash, amount, extendedTimeLockDe
 
   const { publicKey } = networkAddressFormatter.parse(takerAddress)
 
-  const secondsPerBlock = this.currencyConfig.secondsPerBlock
-
-  if (!secondsPerBlock) {
-    const err = 'secondsPerBlock is not specified in the currencyConfig for lnd-engine'
-    this.logger.error(err)
-    return { permanentError: err }
-  }
+  const secondsPerBlock = this.secondsPerBlock
```

Comment:
since this is internal, i'm still cool with using `this.currencyConfig.secondsPerBlock` unless we need to expose this to the user